### DISCUSSION TITLE
doubt - script

### DISCUSSION THREAD

**Discussion by fabinhodigdin** [bb-Ricardo/netbox-sync#264] [General]
Hello everyone, I'm a beginner and I'm trying to synchronize my vcenter with netbox, I've already filled out the " setting.ini " document and in an attempt to run " netbox-sync.py " I have this log below, if anyone can help I'd appreciate it .

LOG

(venv) vectra@vectra-netbox:/opt$ sudo python3 netbox-sync/netbox-sync.py
Traceback (most recent call last):
  File "netbox-sync/netbox-sync.py", line 25, in <module>
    from module.sources import instantiate_sources
  File "/opt/netbox-sync/module/sources/__init__.py", line 11, in <module>
    from .vmware.connection import VMWareHandler
  File "/opt/netbox-sync/module/sources/vmware/connection.py", line 27, in <module>
    from pyVmomi.VmomiSupport import VmomiJSONEncoder
ImportError: cannot import name 'VmomiJSONEncoder' from 'pyVmomi.VmomiSupport' (/usr/lib/python3/dist-packages/pyVmomi/VmomiSupport.py)

**Comment by bb-Ricardo** [comment]
Hi,

the installation does not seem to work as intended.

Did you follow the install steps in README.md? Also there is no need to run this script with root privileges. try starting it without `sudo`.
